lib/features/mailbox/domain/constants/mailbox_constants.dart

```diff
@@ -25,7 +25,7 @@ class MailboxConstants {
     PresentationMailbox.roleDrafts,
     PresentationMailbox.roleSent,
     PresentationMailbox.roleTrash,
-    PresentationMailbox.roleSpam,
+    PresentationMailbox.roleJunk,
```

Comment:
why not keep both here? @dab246

Reply:
We only use the roles set from the server. Using 2 of them leads to the creation of 2 spam folders.

Reply:
in case of no `spam` or `junk`, we will create `junk`

Reply:
> in case of no `spam` or `junk`, we will create `junk`

That's right